elasticdl/python/master/main.py

```diff
@@ -6,8 +8,8 @@ def main():
     args = parse_master_args()
     master = Master(args)
     master.prepare()
-    master.run()
+    return master.run()
```

Comment:
Why not exit directly from `master.run()`? Is this a better practice?

Reply:
I think they are similar.

Reply:
What's the returned value used for?

Reply:
To tell whether this command executes successfully or not.